packages/usehooks-ts/src/useDocumentTitle/useDocumentTitle.md

```diff
@@ -1 +1,3 @@
 An easy way to set the title of the current document.
+
+Setting `prevailOnUnmount` to `false` allows the document title to be reset to its default value (defined by the <title> tag) when the component is unmounted.
```

Comment:
We will prefer `preserveTitleOnUnmount` over `prevailOnUnmount` to be a bit more explicit.